src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/DocumentTypeConverter.java

```diff
@@ -1,16 +1,28 @@
 package uk.gov.hmcts.reform.bulkscanprocessor.entity;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import javax.persistence.AttributeConverter;
 
 public class DocumentTypeConverter implements AttributeConverter<DocumentType, String> {
 
+    private static final Logger log = LoggerFactory.getLogger(DocumentTypeConverter.class);
+
     @Override
     public String convertToDatabaseColumn(DocumentType attribute) {
         return attribute.name().toLowerCase();
     }
 
     @Override
     public DocumentType convertToEntityAttribute(String dbData) {
-        return DocumentType.valueOf(dbData.toUpperCase());
+        try {
+            return DocumentType.valueOf(dbData.toUpperCase());
+        } catch (IllegalArgumentException exception) {
+            // for backwards compatibility due to a lot of incorrect data in db
+            log.warn("Invalid document type found in DB: '{}'", dbData, exception);
```

Comment:
I think we should write a migration script fixing DB data, but then we should always fail when it's invalid.

Reply:
Agree, no reason to keep and support this invalid data.